### PR TITLE
Identity config topic updates

### LINKED ISSUE #5516: Identity config topic updates
security/authentication/identity-configuration.md
https://docs.microsoft.com/aspnet/core/security/authentication/identity-configuration

The topic requires a few minor updates:

* Code-fenced URL segments and property values
* Table work
* Update/add a couple of defaults
* Bust out some API doc links
* Grammar+style updates

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #5516 

[Internal Review Topic](https://review.docs.microsoft.com/en-us/aspnet/core/security/authentication/identity-configuration?branch=pr-en-us-5524)

UE pass:
* Code-fenced URL segments and property values
* Table layout work
* Update/add a couple of defaults
* Bust out some API doc links
* Grammar+style updates

Content hasn't been added/updated, so I don't think we need a technical review on this one.

### CODE REVIEW

File: aspnetcore/security/authentication/identity-configuration.md [hunk 1]
```diff
@@ -13,15 +13,42 @@ uid: security/authentication/identity-configuration
...
 # Configure Identity
 
-ASP.NET Core Identity has common behaviors in applications such as password policy, lockout time, and cookie settings that you can override easily in your application's `Startup` class.
+ASP.NET Core Identity uses default configuration for settings such as password policy, lockout time, and cookie settings that can be overridden in the app's `Startup` class.
 
-## Passwords policy
+## Identity options
 
-By default, Identity requires that passwords contain an uppercase character, lowercase character, a digit, and a non-alphanumeric character. There are also some other restrictions. To simplify password restrictions, modify the `ConfigureServices` method of the `Startup` class of your application.
+The [IdentityOptions](/dotnet/api/microsoft.aspnetcore.identity.identityoptions) class represents the options that can be used to configure the identity system.
```
Comment (scottaddie) [inline aspnetcore/security/authentication/identity-configuration.md]: identity --> Identity

File: aspnetcore/security/authentication/identity-configuration.md [hunk 2]
```diff
@@ -13,15 +13,42 @@ uid: security/authentication/identity-configuration
 
 # Configure Identity
 
-ASP.NET Core Identity has common behaviors in applications such as password policy, lockout time, and cookie settings that you can override easily in your application's `Startup` class.
+ASP.NET Core Identity uses default configuration for settings such as password policy, lockout time, and cookie settings that can be overridden in the app's `Startup` class.
```
Comment (scottaddie) [inline aspnetcore/security/authentication/identity-configuration.md]: I suggest breaking this into 2 sentences

File: aspnetcore/security/authentication/identity-configuration.md [hunk 3]
```diff
@@ -31,61 +58,58 @@ ASP.NET Core 2.0 added the `RequiredUniqueChars` property. Otherwise, the option
...
+| [ChangeEmailTokenProvider](/dotnet/api/microsoft.aspnetcore.identity.tokenoptions.changeemailtokenprovider) | Gets or sets the `ChangeEmailTokenProvider` used to generate tokens used in email change confirmation emails. |
+| [ChangePhoneNumberTokenProvider](/dotnet/api/microsoft.aspnetcore.identity.tokenoptions.changephonenumbertokenprovider) | Gets or sets the `ChangePhoneNumberTokenProvider` used to generate tokens used when changing phone numbers. |
+| [EmailConfirmationTokenProvider](/dotnet/api/microsoft.aspnetcore.identity.tokenoptions.emailconfirmationtokenprovider) | Gets or sets the token provider used to generate tokens used in account confirmation emails. |
+| [PasswordResetTokenProvider](/dotnet/api/microsoft.aspnetcore.identity.tokenoptions.passwordresettokenprovider) | Gets or sets the [IUserTwoFactorTokenProvider<TUser>](/dotnet/api/microsoft.aspnetcore.identity.iusertwofactortokenprovider-1) used to generate tokens used in password reset emails. |
+| [ProviderMap](/dotnet/api/microsoft.aspnetcore.identity.tokenoptions.providermap) | Used to construct UserTokenProviders with the key used as the providerName. |
```
Comment (scottaddie) [inline aspnetcore/security/authentication/identity-configuration.md]: ~~Used to construct UserTokenProviders with the key used as the providerName.~~
Used to construct a `UserTokenProvider` with the key used as the `providerName`.

File: aspnetcore/security/authentication/identity-configuration.md [hunk 4]
```diff
@@ -31,61 +58,58 @@ ASP.NET Core 2.0 added the `RequiredUniqueChars` property. Otherwise, the option
...
 [!code-csharp[Main](identity/sample/src/ASPNETv2-IdentityDemo-Configuration/Startup.cs?range=29-30,48-52)]
 
-`IdentityOptions.User` has the following properties:
+[IdentityOptions.User](/dotnet/api/microsoft.aspnetcore.identity.identityoptions.user) specifies the [UserOptions](/dotnet/api/microsoft.aspnetcore.identity.useroptions) with the properties shown in the table.
 
-| Property                | Description                       | Default |
-| ----------------------- | --------------------------------- | ------- |
-| `RequireUniqueEmail`  | Requires each User to have a unique email. | false  |
-| `AllowedUserNameCharacters`  | Allowed characters in the username. | abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789-._@+ |
+| Property | Description |
```
Comment (scottaddie) [inline aspnetcore/security/authentication/identity-configuration.md]: Add a "Default" column to the table.

Comment (guardrex) [reply aspnetcore/security/authentication/identity-configuration.md]: It was breaking when the doc was built locally (too wide). I can't see the review site anymore, so I'll change it back and you guys can see if it renders ok.

**Comment by guardrex** [comment]
@scottaddie ty ... changes made. :+1:

File: aspnetcore/security/authentication/identity-configuration.md [hunk 5]
```diff
@@ -94,17 +118,35 @@ Under `ConfigureServices` in the `Startup` class, you can configure the applicat
...
+| [ClaimsIssuer](/dotnet/api/microsoft.aspnetcore.authentication.authenticationschemeoptions.claimsissuer) | Gets or sets the issuer that should be used for any claims that are created (inherited from [AuthenticationSchemeOptions](/dotnet/api/microsoft.aspnetcore.authentication.authenticationschemeoptions)). |
+| [Cookie.Domain](/dotnet/api/microsoft.aspnetcore.http.cookiebuilder.domain) | The domain to associate the cookie with. |
+| [Cookie.Expiration](/dotnet/api/microsoft.aspnetcore.http.cookiebuilder.expiration) | Gets or sets the lifespan of a cookie. |
+| [Cookie.HttpOnly](/dotnet/api/microsoft.aspnetcore.http.cookiebuilder.httponly) | Indicates whether a cookie is accessible by client-side script.<br><br>The default value is `true`. |
+| [Cookie.Name](/dotnet/api/microsoft.aspnetcore.http.cookiebuilder.name) | The name of the cookie.<br><br>The default value is `.AspNetCore.Cookies`. |
+| [Cookie.Path](/dotnet/api/microsoft.aspnetcore.http.cookiebuilder.path) | The cookie path. |  |
```
Comment (scottaddie) [inline aspnetcore/security/authentication/identity-configuration.md]: Looks like there's an extra pipe at the end of this line.

Comment (guardrex) [reply aspnetcore/security/authentication/identity-configuration.md]: Great :eye:

**Pull request merged**